studentInfo.py

```diff
@@ -0,0 +1,72 @@
+import random
+def main():
+  students = [
+    Champion()
+  ]
+
+  printHeader()
+  selection = getUserSelection()
+  if selection == 0:
+    RandomizedChampions()
+  else:
+    print ("SELECTION NOT RECOGNIZED")
+
+
+class Champion:
+  lanes = ['mid','adc','support','top','jungle']
```

Comment:
[[PEP 8]](https://www.python.org/dev/peps/pep-0008/)

It's recommended to have whitespace after commas

Example:

```
ice_cream_flavors = ['vanilla', 'strawberry', 'blueberry', 'chocolate']
```